data/assets/scene/solarsystem/missions/apollo/17/bouldersstation6.asset

```diff
@@ -60,46 +44,44 @@ local Station6Frag1Model = {
         Intensity = 0.5
       }
     },
-    PerformShading = false,
-    EnableFaceCulling = false
+    PerformShading = true,
+    AmbientIntensity = 0.4,
+    DiffuseIntensity = 0.0,
```

Comment:
Oh now I understand this after seeing the model again. Since the texture of the boulder already have shadows in it (baked in the images) then it makes no sense to add more from the simulated shading. So this can remain as 0. Same for the others as well.